cluster/ci/origin/config/verification-test.yaml

```diff
@@ -28,6 +28,26 @@ parameters:
 - name: PULL_NUMBER
```

Comment:
This looks a lot like `cluster/ci/origin/config/test.yaml` now

Reply:
Yup, I guess the one difference is the TEST_NAME env var. But it can be defaulted to blank if needed.